File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int singleNum = 0, doubleNum = 0;
        for (int x : nums) {
            if (x <= 9) {
                singleNum += x;
            } else {
                doubleNum += x;
            }
        }
        if (singleNum == doubleNum) {
            return false;
        }
        return true;
    }
};

File: LeetCode/3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        singleNum = 0
        doubleNum = 0
        for x in nums:
            if x <= 9:
                singleNum += x
            else:
                doubleNum += x
        if singleNum == doubleNum:
            return False
        return True